portal-web/test/functional/com/liferay/portalweb/tests/enduser/pageaudit/PageAudit.testcase

```diff
@@ -4386,7 +4386,7 @@ definition {
 		}
 
 		task ("Check if Google PageSpeed Insights tab isn't visible in the 'VIRTUAL INSTANCE SCOPE' section") {
-			AssertElementPresent(
+			AssertElementNotPresent(
```

Comment:
When we perform the refactor of this testcase, we can remove this assert (that isn't at the end of the test only).